frontend/src/metabase/visualizations/visualizations/SmartScalar/SettingsComponents/MenuItem.styled.tsx

```diff
@@ -0,0 +1,17 @@
+import styled from "@emotion/styled";
+import type { MouseEvent } from "react";
+import { Menu } from "metabase/ui";
+
+type MenuItemProps = {
+  onClick?: (e: MouseEvent<HTMLButtonElement>) => void;
+  isSelected?: boolean;
+  py?: string;
+};
```

Comment:
WDYT about using the `aria-selected` prop instead of `isSelected`?

Reply:
Also, I was thinking about using Mantine's `MenuItemProps` type here, so we don't need to declare every prop we want to use inside a styled component. Here's an [example](https://github.com/metabase/metabase/blob/a28d750889e141c02c2862c30066a24453e9730c/frontend/src/metabase/visualizations/visualizations/SmartScalar/SmartScalarSettingsWidgets.styled.tsx#L15) from my branch

Reply:
Thanks for these callouts!